build-tools/packages/build-tools/package.json

```diff
@@ -20,7 +20,7 @@
 	},
 	"scripts": {
 		"build": "concurrently npm:build:compile npm:lint",
-		"build:commonjs": "npm run tsc && npm run build:test",
+		"build:commonjs": "npm run tsc && npm run build:test && copyfiles -f ./src/test/mockPackage.json dist/test",
```

Comment:
I would recommend extracting the copy step into a separate script, Something like `build:copy-test-assets` (we use naming like that elsewhere).

Reply:
I'd also say the extra step should be in the `build:test` script instead of here.